pkg/config/parameter/file/file.go

```diff
@@ -59,8 +62,11 @@ func (f *FileParameter) ResolveValue(context parameter.ResolveContext) (interfac
 	if err != nil {
 		return nil, parameter.NewParameterResolveValueError(context, err.Error())
 	}
+	if f.Escaped {
```

Comment:
just to understand it correctly `Escaped` means that it needs to be escaped? If yes, we should probably rename it because from the naming I would assume that it is already escaped

Reply:
This is a good point: @Laubi , what do you think is the clearest name?

Reply:
Consensus:  let's go with `escape`